### PR TITLE
improve docker dashboard

### PR DESCRIPTION
**Pull request by thibaultlaurens**

### CODE REVIEW

**Comment by thibaultlaurens** [comment]
screen shot 2017-05-22 at 16 50 12

**Comment by alexdias** [comment]
@ThibaultLaurens is it ok for the load to be that high up on the dashboard, or even be a default? Given that it won't work without --net=host...
I'm just worried about users not having --net=host and then saying their docker dashboard isn't working.

**Comment by thibaultlaurens** [comment]
its already there in the current dashboard template but still I think we should show it by default. They can always remove the widget if they don't want to start their containers with `--net=host` I guess

**Comment by alexdias** [comment]
@ThibaultLaurens Ok. Yeah I guess if we make it prominent on the docs that `--net=host -> required for load average` then the users would know

File: docker/dashboards/docker.yaml [hunk 1]
```diff
@@ -28,11 +30,12 @@ positions:
     col: 2
     size_x: 2
     size_y: 2
+    widget_type: number
     series:
       - metric: base.count
         scope:
           tag: docker
-  - title: base.load_1_min
+  - title: load 1
```
Comment (tomashley) [inline docker/dashboards/docker.yaml]: 1 min load avg

File: docker/dashboards/docker.yaml [hunk 2]
```diff
@@ -43,96 +46,103 @@ positions:
...
     col: 1
     size_x: 2
     size_y: 2
+    widget_type: detailed-chart
     series:
       - metric: base.memory
         scope:
           tag: docker
-  - title: base.memory.used
+  - title: memory used
```
Comment (tomashley) [inline docker/dashboards/docker.yaml]: memory used (bytes)

just to be clear when you added (%) to the other widget

File: docker/dashboards/docker.yaml [hunk 3]
```diff
@@ -43,96 +46,103 @@ positions:
...
     col: 3
     size_x: 2
     size_y: 2
+    widget_type: stacked-chart
     series:
       - metric: base.memory.used
         scope:
           tag: docker
-  - title: base.memory.total
+  - title: memory total
```
Comment (tomashley) [inline docker/dashboards/docker.yaml]: memory total (bytes)

**Review by tomashley — COMMENTED** [review]
just some suggestions

**Review by tomashley — APPROVED** [review]

**Pull request merged**